sdk/communication/azure-communication-chat/src/main/java/com/azure/android/communication/chat/signaling/TrouterUtils.java

```diff
@@ -124,7 +136,10 @@ private static BaseEvent getParticipantsRemoved(JSONObject payload) {
                 ChatParticipant chatParticipant = new ChatParticipant();
                 chatParticipant.setUser(communicationUser);
                 chatParticipant.setDisplayName(member.getString("displayName"));
-                chatParticipant.setShareHistoryTime(new Date(member.getString("shareHistoryTime")).toString());
+                chatParticipant.setShareHistoryTime(
+                    OffsetDateTime.ofInstant(
+                        Instant.ofEpochMilli(member.getLong("shareHistoryTime")),
+                        ZoneId.of("UTC")).toString());
```

Comment:
We discussed about deduping the `ChatParticipant` and `ChatParticipantProperties`, good to create a work-item so that we can revisit this.